hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/file/SqlAvroTest.java

```diff
@@ -161,7 +161,7 @@ public void test_schemaDiscovery() {
                         + ", \"timestamp\""
                         + ", \"timestampTz\""
                         + ", \"null\""
-                        + ", object IS NOT NULL"
+                        + ", object"
```

Comment:
Does it somehow relate to the json object change?

Reply:
Not really, while going through JSON changes I have spotted that this particular assert could be improved (similarly to JSON).